solr/core/src/java/org/apache/solr/response/DocsStreamer.java

```diff
@@ -221,6 +222,7 @@ public static Object getValue(SchemaField sf, IndexableField f) {
     KNOWN_TYPES.add(LongPointField.class);
     KNOWN_TYPES.add(DoublePointField.class);
     KNOWN_TYPES.add(FloatPointField.class);
+    KNOWN_TYPES.add(DenseVectorField.class); // DenseVectorField extends FloatPointField
```

Comment:
A nit, but do we need this comment?   Why is it important to know that it extends FloatPointField?